.github/workflows/generate-docs-for-dbt-core.yml

```diff
@@ -19,149 +19,160 @@ defaults:
           ref: main
-          # token: PAT?
+          token: ${{ secrets.FISHTOWN_BOT_PAT }}
  
       - name: Create branch
         run: |
-          git checkout -b ${{needs.set_variables.outputs.branch_name}}
-          git push -u origin ${{needs.set_variables.outputs.branch_name}}
+          git checkout -b ${{ needs.set_variables.outputs.branch_name }}
+          git push -u origin ${{ needs.set_variables.outputs.branch_name }}
       
-      - name: Check current branch
+      - name: Print branch status
         run: |
           git status
 
-  commit_dbt_core:
-    needs: [set_variables, create_dbt_core_branch]
-    runs-on: ubuntu-latest
-
-    steps:
-
-      - name: Check out the dbt-core branch
-        uses: actions/checkout@v3
+      - name: Download index artifact
+        uses: actions/download-artifact@v3
         with:
-          repository: dbt-labs/dbt-core
-          ref: ${{ needs.set_variables.outputs.branch_name }}
-          # token: PAT?
-
-      - uses: actions/download-artifact@v3
-        with:
-          name: ${{needs.set_variables.outputs.index_artifact_name}}
+          name: ${{ needs.set_variables.outputs.index_artifact_name }}
           path: "core/dbt/include"
       
-      - uses: actions/download-artifact@v3
+      - name: Download changelog artifact(s)
+        uses: actions/download-artifact@v3
         with:
-          name: ${{needs.set_variables.outputs.changie_artifact_name}}
+          name: ${{ needs.set_variables.outputs.changie_artifact_name }}
           path: ".changes/unreleased"
 
       - name: Print branch status
         run: |
           git status
 
+      # note that if nothing has changed with the repo, the index file won't be
+      # committed since there are no changes, even if there are changelogs.
       - name: Commit & Push changes
         run: |
           git config user.name 'Github Build Bot'
           git config user.email 'buildbot@fishtownanalytics.com'
           git pull
           git add .
-          git commit -m "Add new index.html and changelog yaml files for dbt-docs"
+          git commit -m "Add new index.html and changelog yaml files from dbt-docs"
           git push
 
       - name: Create Pull Request
-        run:
-          gh pr create --title ${{ needs.set_variables.outputs.pr_title }} \
-                       --body ${{ needs.set_variables.outputs.pr_body }} \
+        run: |
+          gh pr create --title "${{ needs.set_variables.outputs.pr_title }}" \
+                       --body "${{ needs.set_variables.outputs.pr_body }}" \
                        --base main \
-                       --head ${{ needs.set_variables.outputs.branch_name } \
+                       --head ${{ needs.set_variables.outputs.branch_name }} \
                        --label dbt-docs
         env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}  # pretty sure this needs to be a scoped down PAT
+          GITHUB_TOKEN: ${{ secrets.FISHTOWN_BOT_PAT }}
```

Comment:
Is this the scoped down PAT?

Reply:
@leahwicz can you answer this?  I don't have access to it.